docs/source/geo_def.rst

```diff
@@ -89,7 +89,81 @@ based on area extent and a proj4-string or a list of proj4 arguments.
  Area extent: (-5326849.0625, -5326849.0625, 5326849.0625, 5326849.0625)
 
 
-The **load_area** function can be used to parse area definitions from a configuration file. 
+The **load_area** function can be used to parse area definitions from a configuration file.
+Assuming the file **areas.yaml** exists with the following content
+
+.. code-block:: yaml
+
+	ease_sh:
+	  description: Antarctic EASE grid
+	  projection:
+	    a: 6371228.0
+	    units: m
+	    lon_0: 0
+	    proj: laea
+	    lat_0: -90
+	  shape:
+	    height: 425
+	    width: 425
+	  area_extent:
+	    lower_left_xy: [-5326849.0625, -5326849.0625]
+	    upper_right_xy: [5326849.0625, 5326849.0625]
+	    units: m
+	ease_nh:
+	  description: Arctic EASE grid
+	  projection:
+	    a: 6371228.0
+	    units: m
+	    lon_0: 0
+	    proj: laea
+	    lat_0: 90
+	  shape:
+	    height: 425
+	    width: 425
+	  area_extent:
+	    lower_left_xy: [-5326849.0625, -5326849.0625]
+	    upper_right_xy: [5326849.0625, 5326849.0625]
+	    units: m
+
+
+An area definition dict can be read using
+
+.. doctest::
+
+ >>> from pyresample import utils
+ >>> area = utils.load_area('areas.yaml', 'ease_nh')
+ >>> print(area)
+ Area ID: ease_nh
+ Name: Arctic EASE grid
+ Projection ID: None
```

Comment:
Is "Projection ID" kept for compatibility if someone still uses PCS_ID for something?